versions.gradle

```diff
@@ -71,6 +71,11 @@ android_components.concept_fetch = "org.mozilla.components:concept-fetch:$versio
 android_components.lib_fetch = "org.mozilla.components:lib-fetch-httpurlconnection:$versions.android_components"
 android_components.support_rustlog = "org.mozilla.components:support-rustlog:$versions.android_components"
 android_components.support_rusthttp = "org.mozilla.components:support-rusthttp:$versions.android_components"
+android_components.feature_accounts = "org.mozilla.components:feature-accounts:$versions.android_components"
+android_components.concept_engine = "org.mozilla.components:concept-engine:$versions.android_components"
+android_components.support_webextensions = "org.mozilla.components:support-webextensions:$versions.android_components"
+android_components.feature_webcompat = "org.mozilla.components:feature-webcompat:$versions.android_components"
```

Comment:
I don't see the webcompat webext getting installed `WebCompatFeature.install(engine)` was this added by mistake?

Reply:
Removed